website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,64 @@
+
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when existing nodes get full, Kubernetes struggles with automatically provisioning new nodes from a cloud provider.
+
+Karpenter activates when pending pods are unable to be fit onto existing nodes. Karpenter understands the instance types available from cloud providers, such as AWS. Karpenter’s provisioner intelligently selects new instances to start, using rich data from both podspecs and the cloud provider APIs (such as EC2’s flexible fleet API). 
+
+Karpenter balances responding rapidly to un-schedulable pods and making efficient provisioning decision. Karpenter is substantially faster than the cluster autoscaler, while additionally considering podspec labels and cloud provider availability zones. Specifically, the commonly used kubernetes properties such as labels, taints, affinity (node, pod), and anti-affinity are supported.
```

Comment:
I'd avoid the term "rapidly" for reasons mentioned above.

Reply:
I'd also avoid drawing any performance comparisons to the cluster autoscaler in the technical documentation.

Reply:
how about
> Karpenter is more responsive than cluster autoscaler...
?

Reply:
These claims are challenging to back up without rigorous framing and data.

Reply:
I could say:

"If cluster autoscaler isn't meeting your scaling and responsiveness needs, Karpenter may be right for you" 

... but that sounds like a drug commercial. 

I'm willing to change this, but I strongly prefer a concepts doc that gives users an understanding of our intent.